src/main/java/de/mediathekview/mserver/crawler/wdr/parser/WdrTopicOverviewDeserializer.java

```diff
@@ -0,0 +1,86 @@
+package de.mediathekview.mserver.crawler.wdr.parser;
+
+import static de.mediathekview.mserver.base.Consts.ATTRIBUTE_HREF;
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.wdr.WdrConstants;
+import de.mediathekview.mserver.crawler.wdr.WdrTopicUrlDTO;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class WdrTopicOverviewDeserializer extends WdrLetterPageDeserializerBase {
+  private static final Logger LOG = LogManager.getLogger(WdrTopicOverviewDeserializer.class);  
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)